source/trading/UI_elements.rst

```diff
@@ -78,31 +83,36 @@ For custom configuration of the *Account Summary Row* follow the steps below.
    the account level. The account level setting has a higher priority and will be applied if there is a configuration at
    both levels.
 #. Set up ``accountSummaryRowData`` object in the `/state`_ to configure the *Account Summary Row* fields.
-#. In the `/state`_ request in the ``accountSummaryRowData`` object, send the values for *Account Summary Row*. 
+#. In the `/state`_ request in the ``accountSummaryRowData`` object, send the values for *Account Summary Row*.
    The order and size of the sent array must match the order and size of the array specified in the configuration.
 
-If the display area of the *Account Summary Row* is undersized, the elements will be hidden sequentially, 
-starting with the very first. This can happen on low-resolution screens. Therefore, arrange information in order of 
+If the display area of the *Account Summary Row* is undersized, the elements will be hidden sequentially,
+starting with the very first. This can happen on low-resolution screens. Therefore, arrange information in order of
 increasing importance.
 
 Account Summary Tab
 ...................
-The *Account Summary Tab* displays the fields received by `/state`_ request as a set of tables. Only one 
-table is displayed, by default. It uses the fields ``balance``, ``unrealizedPL``, and ``equity`` (if sent, the field 
+
+The *Account Summary Tab* displays the fields received by `/state`_ request as a set of tables. Only one
+table is displayed, by default. It uses the fields ``balance``, ``unrealizedPL``, and ``equity`` (if sent, the field
```

Comment:
- Which field is optional?
- Not sure that a comma is necessary before `by default`.